install/kubernetes/helm/istio-remote/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-multi
```

Comment:
you want `istio-control-plane-service-account` instead of `istio-multi`

Reply:
I changed it the other way. Kept the `istio-multi` SA name to match the docs you'd written and fixed the role binding instead.